README.md

```diff
@@ -1,3 +1,94 @@
 # xdggs
 
-Xarray extension for DGGS
+`xdggs` is an extension for [Xarray](https://xarray.pydata.org/) that provides tools to handle geospatial data using Discrete Global Grid Systems (DGGS). It allows efficient manipulation and analysis of multi-dimensional gridded data within a DGGS framework, facilitating spatial data processing, resampling, and aggregations on both global and regional scales.
+
+## Key Features
+
+- **Seamless Integration with Xarray**: Use `xdggs` alongside Xarray's powerful tools for managing labeled, multi-dimensional data.
+- **Support for DGGS**: Convert geospatial data into DGGS representations, allowing for uniform spatial partitioning of the Earth's surface.
+- **Spatial Resampling**: Resample data on DGGS grids, enabling downscaling or upscaling across multiple resolutions.
+- **DGGS Aggregation**: Perform spatial aggregation of data on DGGS cells.
+- **Efficient Data Management**: Manage large datasets with Xarray's lazy loading, Dask integration, and chunking to optimize performance.
+
+## Installation
+
+To install `xdggs`, you can clone the repository and install it using pip:
+
+```bash
+git clone https://github.com/your-repo/xdggs.git
+cd xdggs
+pip install .
+```
+
+Alternatively, you can install it directly via pip (once it's available on PyPI):
+
+```bash
+pip install xdggs
+```
+
+## Demo
+
+![xdggs demo](xdggs-cropped.gif)
+
+## Getting Started
+
+Here's a quick example of how to use `xdggs` to convert your Xarray data to a DGGS grid and perform resampling.
```

Comment:
I don't think this is the right description, with this we're not converting nor resampling, we simply assign lat / lon coordinates and save it back to disk. Something like this might be better:
```suggestion
As an example, this is how you would use `xdggs` to reconstruct geographical coordinates from the cell ids:
```